### ISSUE TITLE
Tag multiple events in one go (mass tagging)

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#1604]
At the moment tags can be applied to a single event from the event view. It would be nice to be able to tag multiple events with the same tag in one go. Event selection could be based on existing tags, text searches or orgc.

**Comment by morallo** [comment]
I agree it would make information maintenance easier, for example replacing old custom tags with new taxonomies.

**Comment by mokaddem** [comment]
Issue has been raised again in training. Could be integrated while reworking the view

**Comment by github-germ** [comment]
Would like bulk tagging in PyMISP also: https://github.com/MISP/PyMISP/issues/337

**Comment by iglocska** [comment]
> Would like bulk tagging in PyMISP also: [MISP/PyMISP#337](https://github.com/MISP/PyMISP/issues/337)

Don't think that's really related to a change in the view.

**Comment by baphael** [comment]
Hi, would it be realistic to edit the endpoint `tags/attachTagToObject` to accept `{'uuid': uuid_array, 'tag': tag_name, 'local': local}` as POST data, where :
- `uuid_array` would be an array of uuids (strings)
- (`tag` : String) **unchanged**
- (`local` : Boolean) **unchanged**

I presume this could have an impact on the response time depending on the array length...

I have a use case (edgy, for sure, but still) where I need to add a local tag to over 1 million attributes (through PyMISP) and it's taking forever if doing it one by one.

Any ideas ?
